lms/templates/instructor/instructor_dashboard_2/certificate-invalidation.underscore

```diff
@@ -3,8 +3,8 @@
 </p>
 
 <div class="add-certificate-invalidation">
-    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>" aria-describedby='student-user-name-or-email-tip'>
-    <textarea class='notes-field' id="certificate-invalidation-notes" rows="10" placeholder="<%- gettext('Add notes about this learner') %>" aria-describedby='notes-field-tip'></textarea>
+    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>">
```

Comment:
I believe this `label` and `placeholder` are resolved in AC-524, so I'm not messing with them here to avoid conflicts.